Brianubyun/navbar

brianubyun opened the pull request:
![image](https://github.com/user-attachments/assets/38bbef04-3fdf-4db3-8b2b-b2a6953b50e1)

this is how it looks like
- did not add responsive feature so it is fixed for all resolutions 
- just display so navbar doesnt actually work

arieen17 commented:
Good work! If possible could you try to create the logo using png->webp, rather than an SVG? And for the text use the <Link> tag rather than divs, then for the links just write href={"/name of whatever page}. Something similar to this format!

brianubyun commented:
![image](https://github.com/user-attachments/assets/f8cd2b6f-1452-4928-a9f7-24553f2307f5)
this is what it looks like now (not sure about spacing)
- changed text to clickable hyperlinks 
- changed to .webp extension 
- spaced out more but not perfect

arieen17 commented:
Thanks for being so quick with the changes! Don't forget to include the blue line under the red as well, that is a part of the nav (referenced in the Figma). If possible, make the text a bit larger?

brianubyun commented:
![image](https://github.com/user-attachments/assets/f0387779-fd7d-4afd-ac15-f00b4ad71e43)
- added the blue bar
- made the text slightly bigger
- changed some of the code

arieen17 reviewed (approved):
Looks great! Good job :D
